sdk/nodejs/cosmosdb/zMixins.ts

```diff
@@ -0,0 +1,203 @@
+    /**
+     * The name of the database we are subscribing to.
+     */
+    databaseName: pulumi.Input<string>;
+
+    /**
+     * The name of the collection inside the database we are subscribing to.
+     */
+    collectionName: pulumi.Input<string>;
+
+    /**
+     * When set, it customizes the maximum amount of items received per Function call.
+     */
+    maxItemsPerInvocation?: pulumi.Input<number>;
+
+    /**
+     * When set, it tells the Trigger to start reading changes from the beginning of the history of the collection instead of the current time. 
+     * This only works the first time the Trigger starts, as in subsequent runs, the checkpoints are already stored. Setting this to true when 
+     * there are leases already created has no effect.
+     */
+    startFromBeginning?: pulumi.Input<boolean>;
+
+    /**
+     * The resource group in which to create the event subscription.  If not supplied, the Topic's
+     * resource group will be used.
+     */
+    resourceGroup?: core.ResourceGroup;
+
+    /**
+     * The name of the resource group in which to create the event subscription.  If not supplied,
+     * the Topic's resource group will be used.
+     */
+    resourceGroupName?: pulumi.Input<string>;
+
+    /**
+     * Specifies the supported Azure location where the resource exists. Changing this forces a new
+     * resource to be created.  If not supplied, the location of the Topic's ResourceGroup will be
+     * used.
+     */
+    location?: pulumi.Input<string>;
+}>;
+
+declare module "./account" {
+    interface Account {
+        /**
+         * Creates a new subscription to events fired from Cosmos DB Change Feed to the handler provided, along
+         * with options to control the behavior of the subscription.
+         */
+        onChange(
+            name: string, args: CosmosChangeFeedSubscriptionArgs, opts?: pulumi.ComponentResourceOptions): CosmosChangeFeedSubscription;
+    }
+}
+
+Account.prototype.onChange = function(this: Account, name, args, opts) {
+    return new CosmosChangeFeedSubscription(name, this, args, opts);
+}
+
+export class CosmosChangeFeedSubscription extends appservice.EventSubscription<CosmosChangeFeedContext, any[], void> {
+    readonly account: Account;
+
+    constructor(
+        name: string, account: Account,
+        args: CosmosChangeFeedSubscriptionArgs, opts: pulumi.ComponentResourceOptions = {}) {
+
+        opts = { parent: account, ...opts };
+
+        const { resourceGroupName, location } = appservice.getResourceGroupNameAndLocation(args, account.resourceGroupName);
+
+        const bindingConnectionKey = "BindingConnectionAppSettingsKey";
+
+        const bindings: CosmosBindingDefinition[] = [{
+            name: "items",
+            direction: "in",
+            type: "cosmosDBTrigger",
+            leaseCollectionPrefix: name,
+            createLeaseCollectionIfNotExists: true,
```

Comment:
should this be user configurable?

Reply:
Do we want to expose all properties as configurable? (there are more of them) Or take some defaults for now, which make sense in 95% of cases? (my call)

Reply:
we can totally not expose if we think this is reasonable.  users can always request in the future.  though we should likely doc why we think this is the right default.

Reply:
Added a comment to cover both.